services/server/port/createkind411.go

```diff
@@ -0,0 +1,230 @@
+package main
+
+import (
+	"crypto/ed25519"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/HORNET-Storage/hornet-storage/lib/signing"
+	"github.com/HORNET-Storage/hornet-storage/lib/stores"
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/btcec/v2/schnorr"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+	"github.com/joho/godotenv"
+	"github.com/nbd-wtf/go-nostr"
+	"github.com/spf13/viper"
+)
+
+const envFile = ".env"
+const nostrPrivateKeyVar = "NOSTR_PRIVATE_KEY"
+
+type RelayInfo struct {
+	Name          string `json:"name"`
+	Description   string `json:"description,omitempty"`
+	Pubkey        string `json:"pubkey"`
+	Contact       string `json:"contact"`
+	SupportedNIPs []int  `json:"supported_nips"`
+	Software      string `json:"software"`
+	Version       string `json:"version"`
+	DHTkey        string `json:"dhtkey,omitempty"`
+}
+
+func createKind411Event(privateKey *secp256k1.PrivateKey, publicKey *secp256k1.PublicKey, store stores.Store) error {
```

Comment:
We should probably have a place where we describe all non-standard kinds we support but can we have a comment in this file somewhere describing what this kind411 is and why we are overwriting old versions of them?

Reply:
Makes sense.